src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx

```diff
@@ -388,7 +388,7 @@
     <value>False block</value>
   </data>
   <data name="PortDataImageToolTip" xml:space="preserve">
-    <value>image</value>
+    <value>Image for visualization</value>
```

Comment:
I think the same change needs to be made to the `Resources.resx` file.

Reply:
Addressed in b589c6c